modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavascriptClosureAngularClientCodegen.java

```diff
@@ -64,15 +70,11 @@ public JavascriptClosureAngularClientCodegen() {
 
         typeMapping.put("binary", "string");
 
-        outputFolder = "generated-code/javascript-closure-angular";
```

Comment:
I assume this line has to be added in the constructor. And it has to be removed from line 99 and 107
`outputFolder = "generated-code/javascript-closure-angular";`